### ISSUE TITLE
prettier jsdoc: doesn't work with missing space after keyword

### ISSUE THREAD

**Issue by wmertens** [homer0/packages#25]
Another fun one:

```js
const StyledInputLayout = /**
 * @type {import('styled-components').ThemedStyledFunction<
 * 		InputLayout<Input, Props, Value>,
 * 		null,
 * 		{
 * 			inline?: boolean
 * 			fitChildren?: boolean
 * 			together?: boolean
 * 			top?: boolean
 * 			gap?: boolean | string
 * 			right?: boolean
 * 			theme: Object
 * 		}
 * 	>}
 * @template {InputComponent<Props, Value>} Input
 * @template{InputProps<Value>} [Props=React.ComponentProps<Input>]
 * @template [Value=Props['value']]
 */
```
(notice the missing space after `@template` for Props

becomes
```js
const StyledInputLayout = /**
 * @type {import('styled-components').ThemedStyledFunction<
 * 		InputLayout<Input, Props, Value>,
 * 		null,
 * 		{
 * 			inline?: boolean
 * 			fitChildren?: boolean
 * 			together?: boolean
 * 			top?: boolean
 * 			gap?: boolean | string
 * 			right?: boolean
 * 			theme: Object
 * 		}
 * 	>}
 * @template {InputComponent<Props, Value>} Input
 * @template [Value=Props['value']]
 * @template{InputProps<Value>} [Props=React.ComponentProps<Input>]
 */
```
which is of course very different :)

**Comment by homer0** [comment]
Hi @wmertens 👋 !

I'll try and take a look during the weekend :D, but I believe the parse may be having issues with the way you are using `@template`; although it was meant to work like TS generics, they didn't have value, type, nor were optional.

They removed it from [the official docs](https://jsdoc.app), but you can still see the examples in the [TS docs](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template), and my implementation and [tests](https://github.com/homer0/packages/blob/main/packages/public/prettier-plugin-jsdoc/test/e2e/fixtures/example-01.fixture.js) are based on that.

**Comment by homer0** [comment]
Hey 👋 !

I know, I suck at handling the issues, sorry! And I misunderstood the issue when I first read it: even though `@template` is no longer in JSDoc, the issue is not in this package, but on `comment-parser`, and it makes sense, is not a formatter :P.

I added a regex replace before the parser and seems to be working :D.

I'll try to take a look at #24 before pushing the fix.

**Comment by homer0** [comment]
Hi @wmertens 👋 !

I just released `v5.1.0` with a fix for this.
